### PR TITLE
Statically render "chill" loader while waiting for React

### PR DESCRIPTION
**Pull request by inlinestyle**
Addresses the startup portion of https://github.com/popcodeorg/popcode/issues/781

Here's a recording of the standard page load from popcode.org
chillloaderv1

And here's a recording of a page load of popcode.org with a gist querystring (also shows that the pre-existing chill loader is unchanged)
chillloaderv2

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -9,7 +9,10 @@
   </head>
   <body>
     <div id="main">
-      <div class="loading"></div>
+      <div class="pop-throbber">
+        <div class="pop-throbber__message">Loading…</div>
+        <div class="pop-throbber__image--thinking"></div>
```
Comment (outoftime) [inline static/index.html]: This is annoying, but technically BEM requires that any modifier have a corresponding non-modified block/element. So for full compliance this should also have a class `pop-throbber__image`, even if that class isn’t referenced in the CSS. For simplicity, though, I’d suggest just getting rid of the modifier altogether, and making it implicit that the throbber’s image is a thinking-state Pop (which seems reasonable to assume).

File: src/css/application.css [hunk 2]
```diff
@@ -566,7 +566,7 @@ body {
   position: absolute;
   top: 50%;
   left: 50%;
-  height: 20%;
+  height: 50%;
```
Comment (outoftime) [inline src/css/application.css]: Does this have any effect on the way it renders in the Gist import case? (Visual inspection would suggest no, but I can’t figure out why!)

File: src/css/application.css [hunk 3]
```diff
@@ -582,6 +582,11 @@ body {
   color: var(--color-dark-gray);
 }
 
+.pop-throbber__image--thinking {
+  background: url('../images/pop/thinking.svg') center no-repeat;
```
Comment (outoftime) [inline src/css/application.css]: It would be crazy to inline this, right?

Comment (inlinestyle) [inline src/css/application.css]: Not that crazy? Especially if it were done cleanly, like if it was required as a template via ejs or something. I'd save it for a future PR though.

File: src/css/application.css (hunk 2 above)
Comment (inlinestyle) [inline src/css/application.css]: Not that I could tell? I'd manually verified in dev tools that this property had no effect on display for Gist import. Are there other UI configurations that this is used for?

File: static/index.html (hunk 1 above)
Comment (inlinestyle) [inline static/index.html]: Done!

**Pull request merged**